orcestra/postprocess/level1.py

```diff
@@ -148,6 +149,45 @@ def _filter_spikes(ds, threshold=5, window=1200):
     return xr.where(abs(diff) < threshold, ds, interpolated)
 
 
+def _filter_land(ds, mask, lat, lon, offset=pd.Timedelta("7s")):
+    """
+    Filters out data that was collected over land.
+    Removes data by offset earlier than the the time the plane flies over land
+    to avoid including land measurements due to tilt of the plane or mask inaccuracies.
+
+    Parameters
+    ----------
+    ds : xr.Dataset
+        Dataset to filter.
+    mask : xr.DataArray
+        Mask of land and sea. 1 for sea, 0 for land.
```

Comment:
How about naming this variable `sea_land_mask` (as you defined sea as 1)?

Reply:
Sounds good, will implement that.